### PR TITLE
zh-trans: content/zh/docs/reference/glossary/kubelet.md

### PR DESCRIPTION
**Pull request by pigletfly**

### CODE REVIEW

File: content/zh/docs/reference/glossary/kubelet.md [hunk 1]
```diff
@@ -11,9 +25,16 @@ tags:
 - fundamental
 - core-object
 ---
+-->
+<!--
  An agent that runs on each node in the cluster. It makes sure that containers are running in a pod.
+-->
+一个在集群中每个节点上运行的代理。它保证容器都运行在一个 Pod 中。
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/kubelet.md]:
```suggestion
一个在集群中每个节点上运行的代理。它保证容器都运行在 Pod 中。
```

File: content/zh/docs/reference/glossary/kubelet.md [hunk 2]
```diff
@@ -11,9 +25,16 @@ tags:
...
 
 <!--more--> 
 
+<!--
 The kubelet takes a set of PodSpecs that are provided through various mechanisms and ensures that the containers described in those PodSpecs are running and healthy. The kubelet doesn’t manage containers which were not created by Kubernetes.
+-->
 
+kubelet 通过一组 PodSpecs 提供的各种机制，并确保这些 PodSpecs 中描述的容器运行且健康。kubelet 不会管理不是由 Kubernetes 创建的容器。
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/kubelet.md]:
```suggestion
kubelet 接收一组通过各类机制提供给它的 PodSpecs，确保这些 PodSpecs 中描述的容器处于运行状态且健康。kubelet 不会管理不是由 Kubernetes 创建的容器。
```

**Comment by pigletfly** [comment]
Updated @tengqm .Thanks for your reviewing.

**Pull request merged**